proposals/0000-unify-natural.rst

```diff
@@ -0,0 +1,59 @@
+Unify ``Nat`` and ``Natural``
+=============================
+
+.. author:: Richard Eisenberg, inspired by Rinat Striungis
+.. date-accepted:: 
+.. ticket-url:: 
+.. implemented:: 
+.. highlight:: haskell
+.. header:: This proposal is `discussed at this pull request <https://github.com/ghc-proposals/ghc-proposals/pull/364>`_.
+.. contents::
+
+GHC currently uses ``GHC.TypeLits.Nat`` to describe compile-time natural numbers
+but ``Numeric.Natural.Natural`` to describe runtime ones. This proposal unifies
+the two by making ``GHC.TypeLits.Nat`` a synonym for ``Numeric.Natural.Natural``.
+
+Motivation
+----------
+GHC should not have two different types to represent natural numbers. One is
+enough.
+
+Proposed Change Specification
+-----------------------------
+* Add ``type Nat = Natural`` in ``GHC.TypeNats``.
+
+* Re-export ``Natural`` from ``GHC.TypeLits`` (along with the current re-export
```

Comment:
Mention `GHC.TypeNats` as well, since that's where the definition is `Nat` is. (`GHC.TypeLits` just re-exports it.)